.docker/mujoco_installer.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+rm -rf "${HOME}/mujoco"
+
+for mj_version in $MUJOCO_VERSIONS
+do
+	echo "Installing MuJoCo Version ${mj_version} ..."
+	curl -O -L https://github.com/google-deepmind/mujoco/releases/download/${mj_version}/mujoco-${mj_version}-linux-x86_64.tar.gz || echo "could not find version ${mj_version}!"
```

Comment:
This should be preceded by 

```bash
available=$(curl -o /dev/null --silent -Iw '%{http_code}' \
		https://github.com/google-deepmind/mujoco/releases/download/${mj_version}/mujoco-${mj_version}-linux-x86_64.tar.gz)
	if [[ $available -eq 404 ]]; then
		echo "version ${mj_version} could not be found!"
		continue
	fi
```

Otherwise, non-existing version numbers are redirected to a 404 page stating "Not Found" which is still downloaded and will cause

Reply:
Better use option -f (--fail).